server/graphql/types/src/types/temperature_breach.rs

```diff
@@ -165,6 +165,9 @@ impl TemperatureBreachNodeType {
             from::ColdCumulative => to::ColdCumulative,
             from::HotConsecutive => to::HotConsecutive,
             from::HotCumulative => to::HotCumulative,
+            from::Excursion => {
```

Comment:
yes, you may say that I'm shoe horning the notification config into this table, which causes problems like this.. but I think the mix of Excursion and Breach here is sort of valid, and it does make the configuration much neater without adding overhead.